portfolio/src/main/webapp/comments.html

```diff
@@ -32,17 +32,27 @@
         <h1>Comments</h1>
       </div>
       <form action="/data" method="POST">
-        <!-- TODO(aabundis): Add pseudonym feature to identify owner of comment. -->
+        <h3 class="subtitle">Name:</h3>
+        <div class="row">
+          <div class="col">
+            <input type="text" class="form-control" placeholder="First name" name="firstName">
+          </div>
+          <div class="col">
+            <input type="text" class="form-control" placeholder="Last name" name="lastName">
+          </div>
+        </div>
         <div class="form-group">
           <label for="textarea"><h3 class="subtitle">Leave your comment:</h3></label>
           <textarea class="form-control" name="textarea" rows="3"></textarea>
         </div>
         <button type="submit" class="btn btn-light">Submit</button>
       </form>
+      <!-- To see admin page, uncomment next line. -->
+      <!-- <a href="/_ah/admin" id="text">View admin page.</a> -->
```

Comment:
Is this a TODO?

Reply:
It's more of a tool for when I was testing in the dev server, if clicked it'd to see the stored data in the dev server (different form the deployed app's server). Would you consider it better for me to remove it from the code or leave it there if further coding and debugging is required?

Reply:
Ah okay that's fine! You can probably leave it as is. If you would like you could add to the comment explaining that it is for dev purposes.

Reply:
Done!